alpha/apps/kaltura/lib/storage/kFileSyncUtils.class.php

```diff
@@ -1117,6 +1117,11 @@ public static function createSyncFileForKey ( $rootPath, $filePath, FileSyncKey
 		$currentDCFileSync->setIsDir($isDir);
 		$currentDCFileSync->save();
 
+		if ($currentDCFileSync->getObjectType() == FileSyncObjectType::ASSET && $currentDCFileSync->getObjectSubType() == asset::FILE_SYNC_ASSET_SUB_TYPE_ASSET)
+		{
+			self::setActualFileSizeOnAsset($currentDCFileSync);
```

Comment:
No need to limit it only to sub type asset, lets do it for all assets.
For all asset , we have new property called sizeBytes, and it is the actual files on disk.